### DISCUSSION TITLE
OpenRPC dynamically render spec

### DISCUSSION THREAD

**Discussion by hholst80** [bcb/jsonrpcserver#151] [Ideas]
This page can be pointed to any OpenRPC schema:
https://playground.open-rpc.org/

It looks pretty professional to have a spec for the API and having the server component generate it for me would be great. I think FastAPI can generate OpenAPI: https://fastapi-restful.netlify.app/user-guide/openapi/ This feature would be the equivalent for OpenRPC (JSON-RPC 2.0)

**Comment by bcb** [comment]
Sounds good. I think we can populate most of that by analysing the methods.
